Makefile

```diff
@@ -153,6 +154,17 @@ install:
 install-statik:
 	go get -u github.com/rakyll/statik
 
+.PHONY: install-go-bindata-assetfs
+install-go-bindata-assetfs:
+	go get -u github.com/jteeuwen/go-bindata/...
+	go get -u github.com/elazarl/go-bindata-assetfs/...
+
+.PHONE: statik-elasticsearch-mappings
```

Comment:
Should this be changed to remove `statik` from the task?

Reply:
will be changed in a follow pr with the unification of statik and go-bindata